articles/tables.md

```diff
@@ -681,6 +692,35 @@ Clicking on any file will download it.
 {% endtab %}
 {% endtabs %}
 
+## Table Facets
+The faceted navigation on `Tables` can be used to help simplify your search without having to use SQL-like queries. To use table facets, navigate to a `Table` or a `File View`. For this example, we'll be using the [Synapse Table Demo](https://www.synapse.org/#!Synapse:syn3079449/tables/){:target="_blank"} found in the [Wondrous Research Example](https://www.synapse.org/#!Synapse:syn1901847/wiki/56044){:target="_blank"}.
+
+### Set facets
+In order to use simple search, you must first set columns to be facets in the schema editor. Select **Schema** in the upper right of your table and click on **Edit Schema**. In the resulting pop-up, select **Values** or **Range** from the dropdowns under the **Facet** option.
```

Comment:
define 'value' vs 'range' - which column types can be used for each - what to expect